src/asset_util/src/lib.rs

```diff
@@ -28,9 +28,17 @@ pub const IC_CERTIFICATE_EXPRESSION: &str =
 /// certification to be valid.
 #[derive(Debug, Default, Clone)]
 pub struct CertifiedAssets {
```

Comment:
nit: maybe rename to `CertifiedAssetsStore` or something similar, as otherwise it is slightly confusing that we have two structs `CertifiedAssets` and `CertifiedAsset` and the former is not a container of the latter (in the definition of the structs).

Reply:
Well semantically it does. The crate does not give visibility into the internals of `CertifiedAssets` (and a consumer shouldn't care). But you can retrieve a `CertifiedAsset` from `CertifiedAssets` using the `certified_asset` function. So all is as expected?